src/Umbraco.Infrastructure/PropertyEditors/RichTextEditorPastedImages.cs

```diff
@@ -84,7 +84,7 @@ public string FindAndPersistPastedTempImages(string html, Guid mediaParentFolder
                 continue;
             }
 
-            var absoluteTempImagePath = _hostingEnvironment.MapPathContentRoot(tmpImgPath);
+            var absoluteTempImagePath = _hostingEnvironment.MapPathWebRoot(tmpImgPath);
```

Comment:
This seems wrong.. I guess there is another place that uses WebRoot, that should instead use ContentRoot

Reply:
Seems to work if we use CotnentRoot in TinyMceController.UploadImage

Reply:
I will fix that and push